### ISSUE TITLE
Double scrollbars atop one another

### ISSUE THREAD

**Issue by NeighNeighNeigh** [deathau/Notation-for-Obsidian#3]
Getting a weird double scrollbar in a note window. I have disabled all other snippets to make sure there wasn't a conflict. If I move the panel to a different part of the UI the scrollbar behaves correctly. [See here for a screen cap of the issue.](https://www.dropbox.com/s/wktyrgx9b116h4f/Screen%20Recording%202020-12-11%20at%205.38.57%20pm%20sm.mov?dl=0)

Edit. It seems to have disappeared. Im not confident it wasn't user error.

**Comment by NeighNeighNeigh** [comment]
Actually yeah, the issue came back after switching from preview back to edit mode :(

**Comment by deathau** [comment]
Yeah, I think a recent update to Obsidian messed this up somehow. I've been trying to figure out the cause and fix it, but I've been having trouble with it.

It's due to the "in document header" code.

**Comment by NeighNeighNeigh** [comment]
Okay thanks for confirming it’s indeed an issue. With some more poking around, I found the in document header was acting up in other ways too - that is: only editable when in preview mode. I really love the in document header btw :)

**Comment by davingreen** [comment]
Anybody figured out a workaround?
